lib/app.js

```diff
@@ -1,13 +1,13 @@
 import { program } from "commander";
 import { builder } from "./builder.js";
-import { VERSION } from "./version.js"
+import { VERSION, SPEC_VERSION } from "./version.js"
 
 // :: go!
 
 function main() {
  program
    .usage('<command>')
-   .version(VERSION, "-v, --version")
+   .version(`base16-builder v${VERSION}\nBase16 spec v${SPEC_VERSION}`, "-v, --version")
```

Comment:
Not sure if you got the idea from the Go builder or you came up with it on your own, but this is definitely a good idea.

Reply:
Stole it from you. 🥸